tutorials/s4sdkjs-getting-started/s4sdkjs-getting-started.md

```diff
@@ -19,34 +19,36 @@ primary_tag: products>sap-s-4hana-cloud-sdk
 
 [ACCORDION-BEGIN [Step 1: ](Scaffold an application)]
 
-Personally, we are fans of `TypeScript` and recommend using it for most applications. However, we also offer a version of our application scaffolding for plain `JavaScript`. If a step in a tutorial contains instructions or code that is specific to `TypeScript` or `JavaScript`, you can toggle between whatever you prefer at the top of each step.
+Personally, we are fans of `TypeScript` and recommend using it for most applications. However, if you prefer using plain `JavaScript`, we recommend looking at step 4. If a step in a tutorial contains instructions or code that is specific to `TypeScript` or `JavaScript`, you can toggle between whatever you prefer at the top of each step.
 
 The main differences you will notice between `TypeScript` and `JavaScript` are the type annotations and module definitions - ES6 modules in `TypeScript` vs. `commonJS` modules in `JavaScript`. To migrate a `TypeScript` file to `JavaScript`, you only need to change the file extension from `.ts` to `.js`, remove all type annotations and change the `import`s and `export`s.
 
-[OPTION BEGIN [TypeScript]]
-
-To create an `Express.js` application that already contains all the files and configuration you need to use the SAP Cloud SDK for JavaScript, simply clone our TypeScript scaffolding application as follows.
+To create an application that already contains all the files and configuration you need to use the SAP Cloud SDK for JavaScript, you can use SDK's command line interface (CLI). To get the CLI, run the following command:
 
 ```Shell
-git clone --single-branch --branch scaffolding-ts --origin scaffolding https://github.com/SAP/cloud-s4-sdk-examples.git <path/to/your/project>
+npm install -g @sap-cloud-sdk/cli
 ```
 
-[OPTION END]
-
-[OPTION BEGIN [JavaScript]]
-
-To create an `Express.js` application that already contains all the files and configuration you need to use the SAP Cloud SDK for JavaScript, simply clone our JavaScript scaffolding application as follows.
+This will install the CLI globally on your machine, allowing you to use it anywhere.
+Now you can create a new project by running the CLI's `init` command:
 
 ```Shell
-git clone --single-branch --branch scaffolding-js --origin scaffolding https://github.com/SAP/cloud-s4-sdk-examples.git <path/to/your/project>
+sap-cloud-sdk init my-sdk-project
 ```
 
-[OPTION END]
-
-and enter the freshly cloned project:
+If your folder is empty, this will create a new `nest.js` application from scratch. During the process, the CLI will ask to you provide some details, like your project's name. Since this will already install all the necessary dependencies, this might take a minute. If everything worked correctly, you should see output like this:
```

Comment:
Mention to answer "Should a new `nest.js` project be initialized in this folder?" with `y`, and accept the suggested project name.

Reply:
Mention that this may take a while for npm install etc